app/controllers/api/v1/users/devices_controller.rb

```diff
@@ -1,13 +1,17 @@
 class Api::V1::Users::DevicesController < Api::ApiController
   respond_to :json
 
-  acts_as_token_authentication_handler_for User, only: [:update, :create]
+  acts_as_token_authentication_handler_for User, only: %i(update create)
```

Comment:
PR in general is fine. I think the %i constructor is actually a bit less readable though since it's more uncommon. Don't really gain anything when there's 2 values?